Foundation/NSSwiftRuntime.swift

```diff
@@ -14,10 +14,16 @@ import CoreFoundation
 // This mimics the behavior of the swift sdk overlay on Darwin
 #if os(OSX) || os(iOS)
 @_exported import Darwin
-#elseif os(Linux)
+#elseif os(Linux) //// || os(Android) // Causes ambiguity in NSScanner.swift
```

Comment:
I would like a further explanation of the comment, and/or a bug to go with it and noted here, so that future maintainers can understand why this is commented out.